x-pack/plugins/endpoint/public/applications/endpoint/view/hosts/details/index.tsx

```diff
@@ -136,7 +140,12 @@ const PolicyResponseFlyoutPanel = memo<{
         </EuiTitle>
       </FlyoutSubHeader>
       <EuiFlyoutBody data-test-subj="hostDetailsPolicyResponseFlyoutBody">
-        <PolicyResponse />
+        {responseConfig !== undefined && responseActionStatus !== undefined && (
```

Comment:
I'm assuming you will code something to show here when there is no policy response? either due to no message having been received (yet) or an api error condition. (I'm ok if its done as part of a subsequent PR - maybe when we connect up to the API)